docs/book/src/cronjob-tutorial/testdata/project/config/certmanager/kustomization.yaml

```diff
@@ -1,5 +1,7 @@
 resources:
-- certificate.yaml
+- issuer.yaml
+- certificate-webhook.yaml
+- certificate-metrics.yaml
```

Comment:
Open question: Should we have one cert for each?
We might want to simplify and have one cert per project